Update nba-game-viewer extension

gohadar opened the pull request:
## Description

This PR adds a new play-by-play view to completed games and an additional `details` view for games which show some information on the side. The `details` default setting can be toggled in the preferences of the extension or can be toggled temporarily using an action (`⌘D`).

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

<img width="768" alt="image" src="https://github.com/user-attachments/assets/d8c60448-65c9-459e-ab62-b51ea0ae431b" />

<img width="775" alt="image" src="https://github.com/user-attachments/assets/9a3fb2a5-544e-435e-960f-3f8d9af4bf38" />

<img width="771" alt="image" src="https://github.com/user-attachments/assets/b60f3524-4cb9-4a6b-b43b-fd9db4e52ae2" />

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

degouville commented:
Awesome additions @gohadar !

What are those?
![image](https://github.com/user-attachments/assets/c4ad9031-e33e-43cf-85d7-4a3abe48ca72)

djpowers commented:
Looks very good! Nice job @gohadar!

> Awesome additions @gohadar !
> 
> What are those?

I believe those are total record, home record, away/road record. Wasn't sure if this is already doing so at a glance, but including a tooltip on hover would help to identify each one.

gohadar commented:
> Awesome additions @gohadar !
> 
> What are those? ![image](https://private-user-images.githubusercontent.com/6653988/407728511-c4ad9031-e33e-43cf-85d7-4a3abe48ca72.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzgxNTY0MTUsIm5iZiI6MTczODE1NjExNSwicGF0aCI6Ii82NjUzOTg4LzQwNzcyODUxMS1jNGFkOTAzMS1lMzNlLTQzY2YtODVkNy00YTNhYmU0OGNhNzIuanBlZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTI5VDEzMDgzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ2NWQ0YjQ1ZTg1ZTk0Nzg1OWFmNWY3NDUwMDJiZDViMzg3NGNiYWFjOGQ4M2I2NGUzODcwZjY1MjM5ZWRiZWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.NHXdIqyoc2M8TslTng54dZLLO4xgXAGw5jQiA9SKkTk)

Those are the W-L records for each team, the ‘leaderboard’ icon represents the overall record, the ‘home’ icon represents the home games record and the ‘map’ icon represents the away games record.

But I understand that this may not be very clear, do you have any idea on how I can improve that piece of information?

gohadar commented:
> Looks very good! Nice job @gohadar!
> 
> > Awesome additions @gohadar !
> > What are those?
> 
> I believe those are total record, home record, away/road record. Wasn't sure if this is already doing so at a glance, but including a tooltip on hover would help to identify each one.

Yes! I really wanted to add a tooltip on hover but it looks like raycast doesn't allow adding a tooltip to the sidebar metadata. Any other suggestion on how to achieve that?

djpowers commented:
Ah good catch, didn't realize that wasn't available. I don't think leaving as is would be the worst thing. Otherwise my quick idea would be to change the "Records" tag list title to something like  "Record (overall / home / away)."

degouville commented:
> I believe those are total record, home record, away/road record.

Oh it makes sense now thanks ! 

> how I can improve that piece of information

I guess maybe a plane for game away instead of a map could be slightly better

> quick idea would be to change the "Records" tag list title to something like "Record (overall / home / away)."

Yes it would be definitely better IMO

pernielsentikaer commented:
Awesome work, will await corrections and last reviews

gohadar commented:
Thanks for the suggestions guys!

<img width="763" alt="image" src="https://github.com/user-attachments/assets/5d257c73-3d86-46ad-aa9f-607296f6c210" />

I changed the `Records` title to include the description as suggested and changed the away icon to `Icon.AirplaneTakeoff`.
Do you have any other suggestions @djpowers @degouville?

degouville commented:
LGTM ✅

@gohadar awesome! Thanks again

jamtur01 commented:
Nice addition! Thanks @gohadar 👍🏻
